ClientLib/src/main/java/com/o3dr/services/android/lib/drone/mission/MissionItemType.java

```diff
@@ -259,4 +278,12 @@ public void storeMissionItem(MissionItem missionItem, Bundle bundle) {
         return missionItem;
     }
 
+    /**
+     * Indicates if the mission item is supported on the given vehicle type.
+     * @param vehicleType Vehicle type to check against (i.e: plane, copter, rover...)
+     * @return true the mission item is supported, false otherwise.
+     */
+    public boolean isTypeSupported(Type vehicleType){
```

Comment:
@daimonkor same comment as above. If the argument is `null`, the function should return `false`.